packages/after/lib/loadInitialProps.js

```diff
@@ -1,23 +1,19 @@
 import { matchPath } from 'react-router-dom';
 
 export default function loadInitialProps(routes, pathname, ctx) {
-  const promises = [];
-  routes.some(route => {
+  return Promise.all(routes.filter(route => {
     const match = matchPath(pathname, route);
-    if (match && route.component.getInitialProps) {
-      promises.push(
-        route.component.load
-          ? route.component
-              .load() // load it as well
-              .then(() =>
-                route.component
-                  .getInitialProps({ match, ...ctx })
-                  .catch(() => {})
-              )
-          : route.component.getInitialProps({ match, ...ctx }).catch(() => {})
-      );
-    }
-    return !!match;
-  });
-  return Promise.all(promises);
+    if (!match || !route.component.getInitialProps) return false
+    return true
```

Comment:
We could make this one line?

Reply:
could do: `if (match && route.component.getInitialProps) return true` and thats it i think. should work?

Reply:
`routes.filter(route =>  matchPath(pathname, route) && route.component.getInitialProps)` 🚲

Reply:
haha yes 👍